src/popup/components/AccountDetailsBase.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div class="account-details">
+  <div
+    class="account-details"
+    :data-account-address="activeAccount.address"
```

Comment:
Why isn't it a `data-cy`?

Reply:
It could be, but given that it is not an id I think it's better to give it a more descriptive name